charts/fybrik/files/taxonomy/fybrik_module.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "fybrik_module.json",
+    "title": "FybrikModule taxonomy",
+    "type": "object",
+    "description": "Values contributed to by Fybrik modules.",
+    "properties": {
+        "spec": {
+            "type": "object",
+            "properties": {
+                "capabilities": {
+                    "type": "array",
+                    "items": {"$ref": "taxonomy.json#/definitions/Capability"}
+                }
```

Comment:
Shouldn't this include the plugin type as well?

Reply:
@simanadler can you explain what should be defined in the CRD and what should be defined in the taxonomy.
Keep in mind that the taxonomy target cross components definitions., where is the plug-in used elsewhere?